### PR TITLE
docs: Improve Local LLM information and add WSL Troubleshooting

### PR DESCRIPTION
**Pull request by sanegaming**
**Please describe the purpose of this pull request.**
Update Docs for Local LLMs + WSL

**How to test**
Review the .md's

**Have you tested this PR?**
No, I have not verified the WSL Steps myself.

**Related issues or PRs**
N/a

**Is your PR over 500 lines of code?**
No

**Additional context**
N/a

### CODE REVIEW

File: docs/local_llm.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ category: 6580da9a40bb410016b8b0c3
...
+> If you're using Windows and are trying to get MemGPT with local LLMs setup, we recommend using Anaconda Shell, or WSL (for more advanced users). See more Windows installation tips [here](local_llm_faq).
 
 > ⚠️ MemGPT + open LLM failure cases
 >
-> When using open LLMs with MemGPT, **the main failure case will be your LLM outputting a string that cannot be understood by MemGPT**. MemGPT uses function calling to manage memory (eg `edit_core_memory(...)` and interact with the user (`send_message(...)`), so your LLM needs generate outputs that can be parsed into MemGPT function calls.
->
-> Make sure to check the [local LLM troubleshooting page](local_llm_faq) to see common issues before raising a new issue or posting on Discord.
+> When using open LLMs with MemGPT, **the main failure case will be your LLM outputting a string that cannot be understood by MemGPT**. MemGPT uses function calling to manage memory (eg `edit_core_memory(...)` and interact with the user (`send_message(...)`), so your LLM needs generate outputs that can be parsed into MemGPT function calls. See [LINK TO FAQ] for more information.
```
Comment (cpacker) [inline docs/local_llm.md:17]: can you change `[LINK TO FAQ]` to `[the local LLM troubleshooting page](local_llm_faq)`?

File: docs/local_llm_faq.md [hunk 2]
```diff
@@ -71,3 +71,19 @@ This string is not correct JSON - it is missing closing brackets and has a stray
 ### "Got back an empty response string from ..."
 
 MemGPT asked the server to run the LLM, but got back an empty response. Double-check that your server is running properly and has context length set correctly (it should be set to 8k if using Mistral 7B models).
+
+### "Unable to connect to endpoint" using Windows + WSL
+
+>⚠️ We recommend using Anaconda Shell, as WSL has been known to have issues passing network traffic between WSL and the windows host.
```
Comment (cpacker) [inline docs/local_llm_faq.md:77]: `windows host` -> `Windows host`

File: docs/local_llm_faq.md [hunk 3]
```diff
@@ -71,3 +71,19 @@ This string is not correct JSON - it is missing closing brackets and has a stray
...
+If you still would like to try WSL, you must be on WSL version 2.0.5 or above with the installation from the Microsoft Store app.
+You will need to verify your WSL network mode is set to "mirrored"
+
+You can do this by checking the `.wslconfig` file in `%USERPROFILE%'
+
+Add the following if the file does not contain:
+```
+[wsl2]
+networkingMode=mirrored # add this line if the wsl2 section already exists
+```
```
Comment (cpacker) [inline docs/local_llm_faq.md:88]: Can we add a link somewhere in this block to the GitHub issue (that I put in our discord thread)?

File: docs/quickstart.md [hunk 4]
```diff
@@ -5,17 +5,15 @@ category: 6580d34ee5e4d00068bf2a1d
 ---
 
 ### Installation
+> 📘 Using Local LLMs?
```
Comment (cpacker) [inline docs/quickstart.md:8]: Should we move this "Using Local LLMs" block to below the quickstart?

**Review by cpacker — CHANGES_REQUESTED** [review]

File: docs/quickstart.md (hunk 4 above)
Comment (sanegaming) [reply docs/quickstart.md:8]: Hmm, it is also linked in the Advanced section, I think the quick callout here will help users pause if they are using Local LLMs. Maybe move it once MemGPT is more widely used?

**Review by cpacker — APPROVED** [review]

**Pull request merged**